README.md

```diff
@@ -155,6 +156,36 @@ svc/zookeeper-client     ClusterIP   10.31.243.173   <none>        2181/TCP
 svc/zookeeper-headless   ClusterIP   None            <none>        2888/TCP,3888/TCP   2m
 ```
 
+### Deploy a sample Zookeeper cluster with Ephemeral storage
+
+Create a Yaml file called `zk.yaml` with the following content to install a 3-node Zookeeper cluster.
+
+```yaml
+apiVersion: "zookeeper.pravega.io/v1beta1"
+kind: "ZookeeperCluster"
+metadata:
+  name: "example"
+spec:
+  replicas: 3        
+  storageType: ephemeral
+```
+
+```
+$ kubectl create -f zk.yaml
+```
+
+After a couple of minutes, all cluster members should become ready.
+
+```
+$ kubectl get zk
+
+NAME      REPLICAS   READY REPLICAS   VERSION   DESIRED VERSION   INTERNAL ENDPOINT    EXTERNAL ENDPOINT   AGE
+example   3          3                 0.2.7     0.2.7             10.100.200.18:2181   N/A                 94s
+```
+>Note: User should only provide either persistence or ephemeral in the spec, if none of the values is specified default is persistence
```

Comment:
This should be "persistent" and not "persistence" storage.

Reply:
I have rephrased this statement.